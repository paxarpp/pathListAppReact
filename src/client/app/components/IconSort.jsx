import React from "react";

const IconSort = () => (
  <div className="iconSort">
    <div>{String.fromCharCode(9650)}</div>
    <div>{String.fromCharCode(9660)}</div>
  </div>
);
export default IconSort;
