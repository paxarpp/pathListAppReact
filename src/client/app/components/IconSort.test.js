import React from 'react';
import renderer from 'react-test-renderer';
import IconSort from './IconSort';

test('IconSort', () => {
  const component = renderer.create(<IconSort />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
