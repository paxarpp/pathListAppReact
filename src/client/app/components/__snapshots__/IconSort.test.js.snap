// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconSort 1`] = `
<div
  className="iconSort"
>
  <div>
    ▲
  </div>
  <div>
    ▼
  </div>
</div>
`;
